cmake/onnxruntime_python.cmake

```diff
@@ -251,7 +251,7 @@ endif()
 
 if (onnxruntime_USE_NUPHAR)
   file(GLOB onnxruntime_python_nuphar_python_srcs CONFIGURE_DEPENDS
-    "${ONNXRUNTIME_ROOT}/core/providers/nuphar/scripts/*.*"
+    "${ONNXRUNTIME_ROOT}/core/providers/nuphar/scripts/*"
```

Comment:
>${ONNXRUNTIME_ROOT}/core/providers/nuphar/scripts/* [](start = 5, length = 51)

Please note that python wheel only takes .py files, and NUPHAR_CACHE_VERSION may not be packed in wheel.

Reply:
Hmm, what about create_shared.sh and created_shared.cmd?

Reply:
Those are not included in the python wheel.

---
In reply to: [357511621](https://github.com/microsoft/onnxruntime/pull/2646#discussion_r357511621) [](ancestors = 357511621)

Reply:
Seems we don't need to include create_shared.py (and hence NUPHAR_CACHE_VERSION) either?

Reply:
Let me build a python wheel. Just by reading the code, seems to me that all the files under nuphar/scripts are copied into the wheel package.

Reply:
Added a minor fix to include NUPHAR_CACHE_VERSION into the python wheel